src/Workspaces/Core/Portable/Rename/RenameLocation.ReferenceProcessing.cs

```diff
@@ -364,7 +364,12 @@ private static string TrimNameToAfterLastDot(string name)
                     else
                     {
                         // The simple case, so just the single location and we're done
-                        results.Add(new RenameLocation(location.Location, location.Document.Id, isCandidateLocation: location.IsCandidateLocation, isRenamableAccessor: await IsPropertyAccessorOrAnOverride(referencedSymbol, solution, cancellationToken).ConfigureAwait(false)));
+                        results.Add(new RenameLocation(
+                            location.Location, 
+                            location.Document.Id, 
+                            isCandidateLocation: location.IsCandidateLocation, 
+                            isMethodGroupReference: location.IsCandidateLocation && location.CandidateReason == CandidateReason.MemberGroup, 
```

Comment:
Is it worth adding an extension method Location.IsMethodGroupRename()? You do this a lot.